src/main/java/net/consensys/orion/http/handler/privacy/CreatePrivacyGroupHandler.java

```diff
@@ -102,8 +94,10 @@ public void handle(final RoutingContext routingContext) {
 
     final PrivacyGroupPayload privacyGroupPayload = new PrivacyGroupPayload(
         privacyGroupRequest.addresses(),
-        privacyGroupRequest.name(),
-        privacyGroupRequest.description(),
+        privacyGroupRequest.name() == null || privacyGroupRequest.name().isBlank() ? "Default Name"
```

Comment:
If the name is null OR blank, we use the default value. The same applies to the description.